src/utils.ts

```diff
@@ -1,5 +1,11 @@
 const rootFontSize = 16;
-export const convertPxToRem = (pixel: number) => `${pixel / rootFontSize}rem`;
+export const convertPxToRem = (pixel: number, fixedDigits = 3) => {
+  if (!pixel) return '0rem';
+
+  const pixelFixed = Number(pixel.toFixed(fixedDigits));
```

Comment:
why do you need?

Reply:
Line 3 is to prevent generating "0.00rem".
Line 5 is to store the `Number` value since `toFixed` returns a `string`

Reply:
Removed unneeded toFixed 🙏🏻